media/transcoder/create_job_from_preset_batch_mode.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package transcoder
+
+// [START transcoder_create_job_from_preset_batch_mode]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	transcoder "cloud.google.com/go/video/transcoder/apiv1"
+	"cloud.google.com/go/video/transcoder/apiv1/transcoderpb"
+)
+
+// createJobFromPresetBatchMode creates a batch mode job based on a given preset
+// template. See
+// https://cloud.google.com/transcoder/docs/how-to/create-batch-jobs
+// for more information.
+func createJobFromPresetBatchMode(w io.Writer, projectID string, location string, inputURI string, outputURI string, preset string) error {
+	// projectID := "my-project-id"
+	// location := "us-central1"
+	// inputURI := "gs://my-bucket/my-video-file"
+	// outputURI := "gs://my-bucket/my-output-folder/"
+	// preset := "preset/web-hd"
```

Comment:
Preset doesn't seem necessary as an arg given that the test seems to set it to the same thing. Can you remove it and use a local var in the snippet instead?

Reply:
Done